invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -9,21 +9,24 @@
 
 """RDM record schemas."""
 
+from flask_babelex import lazy_gettext as _
 from invenio_drafts_resources.services.records.schema import RecordSchema
-from marshmallow import EXCLUDE, fields, post_dump
+from marshmallow import EXCLUDE, ValidationError, fields, post_dump, validates
 from marshmallow_utils.fields import NestedAttribute
 from marshmallow_utils.permissions import FieldPermissionsMixin
 
 from .access import AccessSchema
 from .metadata import MetadataSchema
 from .parent import RDMParentSchema
-from .pids import PIDSchema
+from .pids import ExternalPIDSchema
 from .versions import VersionsSchema
 
 
 class RDMRecordSchema(RecordSchema, FieldPermissionsMixin):
     """Record schema."""
 
+    TMP_PIDS_TYPE = {"doi", "concep-doi", "oai"}
```

Comment:
Shall we add a comment, e.g. `# PIDS-FIXME` on every thing we need to review and then we add a card to the board to review/find all these comments?
I fear it will be forgotten! :)

Reply:
Done